src/os/bluestore/bluestore_types.h

```diff
@@ -747,9 +747,18 @@ struct bluestore_blob_t {
     }
   }
   /// todo implement me!
```

Comment:
The idea here was to construct a bit mask that will report:
bit 0 : offset, offset+chunk_size                               was used ?
bit 1: offset + chunk_size ... offset + chunk_size*2  was used ?
....
One bit per disk block. `chunk_size` means disk block, or whaterver granularity was requested.

Reply:
yes, in the code chunk_is_unused = (mask & 1) != 0; not calculate offset position bit in unused, just mask &1.